### PR TITLE
Fix Cheetah 2.0 wrong BLTOUCH PWM pin

### PR DESCRIPTION
**Pull request by GerogeFu**
Sorry that i mistake BLTOUCH sig pin PA0 as PWM pin. This is the fix PR. Check the changes.

### CODE REVIEW

File: Marlin/src/pins/stm32f4/pins_FYSETC_CHEETAH_V20.h [hunk 1]
```diff
@@ -60,10 +55,21 @@
...
+#ifndef Z_MIN_PROBE_PIN
+  #define Z_MIN_PROBE_PIN                   PA0
+#endif
+
 //
 // Filament runout
 //
-#define FIL_RUNOUT_PIN                      PB5
+#if NUM_SERVOS
+  #define FIL_RUNOUT_PIN                    PA0
```
Comment (thinkyhead) [inline Marlin/src/pins/stm32f4/pins_FYSETC_CHEETAH_V20.h:69]: It seems strange that the presence of a Servo Probe / BLTouch should force `FIL_RUNOUT_PIN` to conflict with the default `Z_MIN_PROBE_PIN`. Users do have the option use use the `Z_STOP_PIN` for the probe, and in that case there is no reason to apply this logic.

File: Marlin/src/pins/stm32f4/pins_FYSETC_CHEETAH_V20.h [hunk 2]
```diff
@@ -44,14 +44,9 @@
 #endif
 
 //
-// Z Probe
+// Servos
 //
-#if ENABLED(BLTOUCH)
-  #error "You need to set jumper to 5v for Bltouch, then comment out this line to proceed."
```
Comment (thinkyhead) [inline Marlin/src/pins/stm32f4/pins_FYSETC_CHEETAH_V20.h:50]: Why was this condition believed to be needed? Is the 5V jumper not actually needed? Where did the information about a 5V jumper come from originally?

**Comment by thinkyhead** [comment]
Some of these changes might be achieved by adjusting configuration, so let me know if my comments above make sense.

**Comment by thisiskeithb** [comment]
This PR is now severely broken after many force-pushes. Please submit another patch targeting `bugfix-2.1.x`

**Pull request closed without merging**